kit/inputs/src/MaterialTextView/types.ts

```diff
@@ -120,7 +120,8 @@ export type MaterialTextViewTextProps = {
 };
 
 export type MaterialTextViewClearButtonProps = {
-    clear: (() => void) | undefined;
+    clear?: (() => void) | undefined;
+    hiddenButton?: boolean;
```

Comment:
Let's also add a comment why do we need it (I'd also add a TSDoc comment for each prop here)